modules/exploits/windows/http/manageengine_admanager_plus_cve_2023_29084_auth_cmd_injection.rb

```diff
@@ -0,0 +1,221 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
```

Comment:
This isn't necessary any more.

Reply:
Removed the CmdStager include in https://github.com/rapid7/metasploit-framework/pull/18018/commits/ac99cb32711377646d30ce9bee39d1a0dcc6a2f4